native/cocos/renderer/gfx-gles3/GLES3GPUObjects.h

```diff
@@ -279,6 +281,7 @@ struct GLES3GPUShader {
 
     GLES3GPUShaderStageList gpuStages;
     GLuint glProgram = 0;
+    uint32_t hash = 0;
```

Comment:
Can we use `uint64_t hash`? Collision rate of uint32_t is quite high.
It will be even better if we don't use hash as key.
![image](https://user-images.githubusercontent.com/13654656/231039993-2f89fb5f-2536-4807-9d5a-5681b91bda18.png)
https://stackoverflow.com/questions/14210298/probability-of-collision-when-using-a-32-bit-hash

Reply:
I will modify it to be `uint64_t`.